code/src/components/Navbar.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { NavLink } from 'react-router-dom';
+import "../components/Navbar.css"
+import BurgerMenu from './BurgerMenu';
+
+//simple navbar with props
+//by adding 'exact className' the / link isn't active all the time
```

Comment:
Interesting to see how you solved this

Reply:
I have reused the same burger menu that I used for my first project and then in the WeatherApp and now here :)